lightning-invoice/src/lib.rs

```diff
@@ -60,12 +60,12 @@ const MAX_EXPIRY_TIME: u64 = 60 * 60 * 24 * 356;
 /// Default expiry time as defined by [BOLT 11].
 ///
 /// [BOLT 11]: https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md
-const DEFAULT_EXPIRY_TIME: u64 = 3600;
+pub const DEFAULT_EXPIRY_TIME: u64 = 3600;
 
 /// Default minimum final CLTV expiry as defined by [BOLT 11].
 ///
 /// [BOLT 11]: https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md
-const DEFAULT_MIN_FINAL_CLTV_EXPIRY: u64 = 18;
+pub const DEFAULT_MIN_FINAL_CLTV_EXPIRY: u64 = 18;
```

Comment:
Should we change this constant since RL isn't okay with less than `22` (iiuc)?

Reply:
No, the default value is always 18, we require something higher but that means we have to explicitly specify it in the invoice (which we do).

Reply:
Added a comment making this explicit as users are likely to be confused here.